shuup/front/templates/shuup/front/macros/navigation.jinja

```diff
@@ -54,26 +55,23 @@
                 {{ _render_dropdown_item(url("shuup:logout"), "fa fa-sign-out fa-fw", _("Log out"), True) }}
             {% endif %}
         {% endcall %}
-    {% else %}
+    {% elif show_quick_login %} {# Checking if the login dropdown should be removed #}
```

Comment:
_Checking whether the login dropdown should be **shown**_

Reply:
Fixed